src/main.c

```diff
@@ -1113,7 +1113,7 @@ main(int argc, char **argv)
   signal(SIGPIPE, handle_sigpipe); // will be redundant later
   signal(SIGILL, handle_sigill);   // see handler..
 
-  /* Set priviledges */
+  /* Set privileges */
```

Comment:
oops again! my dyslexia combined with English being my non-native language doesn't help.

I recommend to put the two spelling fixes in one commit, the UID fix in a second commit. Personally, I'm okay with having this under the debian package MR, but that's upto @Flole998

Reply:
I don't think this one is yours; I think it was there before.

Reply:
It's a typo I make very often though :)